cmd/api/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/renatoaguimaraes/job-scheduler/internal/worker/api"
+	"github.com/renatoaguimaraes/job-scheduler/pkg/worker/conf"
+)
+
+func main() {
+	config := conf.NewConfig()
+	flag.StringVar(&config.ServerAddress, "host", "localhost:8080", "host:port")
+	flag.StringVar(&config.ClientCA, "ca", "cert/client-ca-cert.pem", "client ca path")
+	flag.StringVar(&config.ServerCertificate, "cert", "cert/server-cert.pem", "server cert path")
+	flag.StringVar(&config.ServerKey, "key", "cert/server-key.pem", "server key path")
+	flag.Parse()
+	api.StartGRPCServer(config)
```

Comment:
Can this return an error?

Reply:
Yes, I have change the func to return error to the caller.